components/brave_wallet_ui/components/desktop/views/portfolio/portfolio-overview.tsx

```diff
@@ -143,14 +144,18 @@ export const PortfolioOverview = () => {
       return a.plus(b)
     })
     return grandTotal.formatAsFiat(defaultCurrencies.fiat)
-  }, [userAssetList, computeFiatAmount])
+  }, [userAssetList, defaultCurrencies, computeFiatAmount])
 
-  const priceHistory = React.useMemo(() => {
-    if (parseFloat(fullPortfolioFiatBalance) === 0) {
+  const isZeroBalance = React.useMemo((): boolean => {
+    return parseFloat(fullPortfolioFiatBalance.replace(/[^0-9.-]+/g, '')) === 0
```

Comment:
Please add a test and comment to describe the regex

Reply:
Moved regex to a `util` with `tests`